minio/kes-docs
# Migration of wiki content for Hugo

This should be a general move of the wiki content to a Hugo docs markdown.
I did some cleanup for steps and consistency with other docs content.

The docs are not staged anywhere.
To preview, follow instructions in the README for building a live preview locally.

Basic steps to preview locally:
- install Hugo 0.110.0 or greater
- From the local directory for the repo, run `npm start`
- See the preview at `localhost:1313` in your browser

Closes #2 .
